<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Algorithm:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>180</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="8">
     <widget class="QLineEdit" name="lineEdit">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Check:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="6">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string notr="true">TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Copyright 2017-2020 DanSoft. All rights reserved.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>140</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Compare the hash</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string notr="true">TextLabel</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
